.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,45 +8,36 @@ Please label this PR with one of the following labels, depending on the scope of
 - Docs
 -->
 
-## What does this PR do?
+## What is the problem this PR solves?
 
-<!-- Mandatory
-Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
--->
+// Please do not just reference an issue. Explain WHAT the problem this PR solves here.
 
-## Why is it important?
+## How does this PR solve the problem?
 
-<!-- Mandatory
-Explain here the WHY, or the rationale/motivation for the changes.
+// Explain HOW you solved the problem in your code. It is possible that during PR reviews this changes and then this section should be updated.
+
+
+## How to test this PR locally
+
+<!-- Recommended
+Explain here how this PR will be tested by the reviewer if anything special is needed for manual testing: commands, dependencies, steps, etc.
 -->
 
 ## Checklist
 
 <!-- Mandatory
-Add a checklist of things that are required to be reviewed in order to have the PR approved
-
-List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply. (Just in case, strikethrough uses two tildes. ~~Scratch this.~~)
+This checklist is to help creators of PRs to find parts which might not be directly related to code change but still need to be addressed. Anything that does not apply to the PR should be removed from the checklist.
 -->
 
-- [ ] My code follows the style guidelines of this project
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have made corresponding change to the default configuration files
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.
+- [ ] Documentations added / updated for the change.
```

Comment:
How is this different from `I have made corresponding changes to the documentation`?

Reply:
It's not 🤦‍♂️  Fixing.